src/ffi/logging.rs

```diff
@@ -328,9 +341,7 @@ mod test {
         }
     }
 
-    #[test]
-    fn test_log_censor() {
-        let examples = [
+    const EXAMPLES: [(&'static str, &'static str); 11] = [
             (
                 "1999-09-09 [INFO] New endpoint (1.2.3.4:1234) created",
                 "1999-09-09 [INFO] New endpoint (IP(959535cab4852bd4):1234) created",
```

Comment:
Silly idea, but what if instead of using some hash values, we had something in a more human friendly form, like the ones [generated by names](https://github.com/fnichol/names)? Would still obfuscate the address, but would be much easier to visually distinguish the devices when reading the logs.

Reply:
Hmm maybe we could have the what3words style with suffix code: `adj-adj-noun-hex`, then we would need 1024 word lists, and a 34bit suffix.

Reply:
Sounds like a possible solution, I will create a ticket for this improvement.